coda/coda_replication/factories.py

```diff
@@ -0,0 +1,23 @@
+"""
+Coda Replication Model factories for test fixtures.
+"""
+from datetime import datetime
+
+import factory
+from factory import fuzzy
```

Comment:
No because we would have to name every import that we need from the `factory` module. The only reason `fuzzy` is imported is because the package requires an explicit import. See [here](https://factoryboy.readthedocs.org/en/latest/fuzzy.html#module-factory.fuzzy)